src/Widgets/ColumnContainerWidget/ColumnsEditorTab.tsx

```diff
@@ -56,14 +63,23 @@ const ColumnsEditor = connect(
           }}
           readOnly={readOnly}
         />
+        <Switch
+          className="two_valued"
+          labels={['Grid', 'Flex']}
+          onClick={() =>
+            widget.update({ layoutMode: isFlex ? 'grid' : 'flex' })
+          }
+          title="Display mode"
```

Comment:
The dialog version uses some "blue" color where mostly only green should be used:

![Screenshot 2024-08-06 at 10 26 41](https://github.com/user-attachments/assets/634596ad-cf40-46fd-a78f-8d21fc6a0bc8)


Compare this to the current `main` version:

![Screenshot 2024-08-06 at 10 26 24](https://github.com/user-attachments/assets/792d3afa-4947-482d-8c41-5f17f2e62114)